integration_tests/CMakeLists.txt

```diff
@@ -219,8 +219,8 @@ RUN(NAME bindc_01            LABELS cpython llvm c)
 RUN(NAME bindc_02            LABELS cpython llvm c)
 RUN(NAME bindc_04            LABELS llvm c)
 RUN(NAME bindc_07            LABELS cpython llvm c)
-RUN(NAME exit_01             LABELS cpython llvm c wasm wasm_x86 wasm_x64)
-RUN(NAME exit_02     FAIL    LABELS cpython llvm c wasm wasm_x86 wasm_x64)
```

Comment:
```
from sys import exit
```
This also imports the `argv` initialization.
As Wasm doesn't support the list type yet, I removed it from here.

Reply:
I think you should not de-register test for certain backends. That would be wrong. Instead create a new one and register it with limited backends.

Reply:
I wouldn't de-register tests. @Shaikh-Ubaid How long would it take to support lists in WASM? Is it even possible to support lists in WASM?

Reply:
>@Shaikh-Ubaid How long would it take to support lists in WASM? Is it even possible to support lists in WASM?

I think supporting lists have some prerequisites like structs/pointers. Thus, I think it could take sometime to reach lists. 

>Is it even possible to support lists in WASM?

Yes, I think it should be possible to support lists in the `wasm` backend.

Reply:
@Thirumalai-Shaktivel Then I would say extract out the parts in `exit_01.py` which are important for LLVM testing and then put it into a new file `exit_0n.py` (don't put `sys` module imports and register this new file with `wasm`, `wasm_x86`, `wasm_x64`. Then you can remove WASM tests from `exit_01.py`. Same for https://github.com/lcompilers/lpython/pull/1679#discussion_r1170041964

Reply:
Done!